frontend/src/types.ts

```diff
@@ -1134,6 +1134,7 @@ export interface MultivariateFlagOptions {
 interface FeatureFlagFilters {
     groups: FeatureFlagGroupType[]
     multivariate: MultivariateFlagOptions | null
+    aggregation_group_type_index?: number | null
```

Comment:
non-blocking

Is it possible to get a comment on this? It seems rather opaque what this does and what the index means. I'm presuming `null | undefined` means the filters relate to users, then 0, 1, 2, etc relate to Organization, <insert other group variants>, ... ?

Reply:
More or less. One additional wrinkle here is release percentage. 

In the backend `aggregation_group_type_index` is used to determine what we'll be aggregating the flag on. We want the experience to be consistent within a group (see https://github.com/PostHog/posthog/issues/7010 for more details on the requirements). This is also why we don't allow mixing group and person properties.